server/service/src/sensor/berlinger.rs

```diff
@@ -17,7 +17,7 @@ use util::uuid::uuid;
 
 use temperature_sensor::*;
 
-pub fn get_breach_row_type(breach_type: &BreachType) -> TemperatureBreachRowType {
+fn get_breach_row_type(breach_type: &BreachType) -> TemperatureBreachRowType {
```

Comment:
I wanted to see what is still used, thus removed `pub` (so that compiler can help)